Owncloud iOs Client/Tabs/FileTab/Share/ShareMainViewController.m

```diff
@@ -353,6 +375,8 @@ - (void) updateInterfaceWithShareLinkStatus {
             self.isExpirationDateEnabled = true;
         }
         
+        self.isAllowEditingShown = [self hasAllowEditingToBeShown];
+        self.isAllowEditingEnabled = [UtilsFramework isPermissionToCanEdit:self.updatedOCShare.permissions];
```

Comment:
Allow public uploads gives 7 permissions. We need to check it without the permissions of delete.
We could rename isPermissionToCanEdit method to a more accurate name.